packages/react-native/src/theme/tokens/index.ts

```diff
@@ -0,0 +1,53 @@
+import { components } from './components';
+import { ComponentStyles } from '../types';
+import { ColorValue } from '@aws-amplify/ui';
+
+interface BaseTokens {
+  colors: Partial<Colors>;
+}
+
+export interface Tokens extends BaseTokens {
+  components: ComponentStyles;
+}
+
+// TODO: remove color types and values and use the output generated by style-dictionary with correct types
+
+declare type ScaleKeys = 10 | 20 | 40 | 60 | 80 | 90 | 100;
+type ColorScale = {
+  [key in ScaleKeys]: ColorValue;
+};
+
+type Colors = {
+  brand: {
+    primary: ColorScale;
+    secondary: ColorScale;
+  };
+};
+
+export const colors: Partial<Colors> = {
+  brand: {
+    primary: {
+      10: 'hsl(190, 75%, 95%)',
+      20: 'hsl(190, 75%, 85%)',
+      40: 'hsl(190, 70%, 70%)',
+      60: 'hsl(190, 50%, 50%)',
+      80: 'hsl(190, 95%, 30%)',
+      90: 'hsl(190, 100%, 20%)',
+      100: 'hsl(190, 100%, 15%)',
+    },
```

Comment:
Can we just import these directly from the `@aws-amplify/ui`?

Reply:
No, colors are not exported. This code will be removed once we generate the non-conponent tokens. This is just an example. I can remove entirely if too confusing.